### PR TITLE
add updated versions

### PR DESCRIPTION
**Pull request by SebastienValla**

### CODE REVIEW

File: commands/init/package.json [hunk 1]
```diff
@@ -21,9 +21,9 @@
     "license": "ISC",
     "devDependencies": {
         "@massalabs/as": "^1.0.2",
-        "@massalabs/as-tester": "^0.0.2-dev",
+        "@massalabs/as-tester": "^0.0.2",
```
Comment (peterjah) [inline commands/init/package.json:24]: you should keep the nightly version here: "^0.0.3-dev"

File: commands/init/package.json [hunk 2]
```diff
@@ -21,9 +21,9 @@
...
         "@massalabs/as-transformer": "^0.0.2-dev",
-        "@massalabs/massa-as-sdk": "^1.1.1-dev",
+        "@massalabs/massa-as-sdk": "^1.1.1",
```
Comment (peterjah) [inline commands/init/package.json:26]: use nightly "^1.1.2-dev"

File: commands/init/package-lock.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "my-massa-sc",
     "version": "0.0.1",
-    "lockfileVersion": 2,
+    "lockfileVersion": 3,
```
Comment (peterjah) [inline commands/init/package-lock.json:4]: why lockfileVersion changes ?? do you use a different version of npm ??

**Review by peterjah — CHANGES_REQUESTED** [review]

**Review by thomas-senechal — CHANGES_REQUESTED** [review]

File: commands/init/README.md [hunk 4]
```diff
@@ -21,44 +21,43 @@ export function HelloWorld(): void {
...
-    ```
-    $ npm run build
-    ```
--   Running the simulate script
-    ```
-    $ npm run simulate
-    ```
+- Compiling the example smart contracts
+
+    npm run build
```
[the remark this answers is not in the document]
Comment (Thykof) [reply commands/init/README.md:56]: you're right, let's keep it

**Review by peterjah — APPROVED** [review]

**Review by thomas-senechal — APPROVED** [review]

**Review by gregLibert — APPROVED** [review]
:man_shrugging: LGTM

**Pull request merged**